src/shared/services/organisation/organisation.service.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { AbstractAppConfig } from '../../../app.config';
+import { Observable } from 'rxjs';
+import { OrganisationService, Organisation, OrganisationAddress } from './organisation.service';
+import { HttpService } from '../../services';
+import createSpyObj = jasmine.createSpyObj;
+
+describe('Organisation Service', () => {
+
+    const PRD_URL = 'http://aggregated.ccd.reform/api/caseshare/orgs';
+
+    let appConfig: any;
+    let httpService: any;
+    let organisationService: OrganisationService;
+
+    let ORGANISATIONADDRESS: OrganisationAddress [] = [{
+            addressLine1: '12',
```

Comment:
Indenting is off here. Actually, the whole file is also using indenting of 4 spaces when it should be 2, which is the agreed standard for the codebase.